src/devices/deviceConnected.js

```diff
@@ -1,23 +1,32 @@
 import Sensor from '../API/sensors/sensor.model';
-import {stream} from '../data-streams/input';
+import input from '../data-streams/input';
 import Debugger from 'debug';
 
-const debug = Debugger('device-connected');
+const debug = Debugger('SH_BE:device-connected');
 
 
-const DEVICE_CONNECTED_EVENT = 'add';
+const DEVICE_INFO_EVENT = 'device-info';
 
 export default function trackDeviceConnection() {
-    stream
-        .filter(message => message.event === DEVICE_CONNECTED_EVENT)
+    input.stream
+        .filter(message => message.event === DEVICE_INFO_EVENT)
         .subscribe(saveDevice, onError);
 
     function saveDevice (data) {
-        Sensor.find({description : data.value.description}, function (error, records) {
+
+        Sensor.find({
+            mqttId : data.device
+        }, function (error, records) {
+            
             if (records.length){
                 debug('Device already added');
             }else{
```

Comment:
fixed, but I think that such conventions must be handled with eslint not manually.